lib/gui/modules/analytics.js

```diff
@@ -39,6 +40,22 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Get OS architecture
+ *
+ * @description
+ * We need this because node's os.arch() returns the process architecture
+ * See: https://github.com/nodejs/node-v0.x-archive/issues/2862
+ *
+ * @return {String} OS architecture
+ */
+const getOSArch = () => {
+  if (os.arch() !== 'ia32') {
```

Comment:
So if the NodeJS architecture is x64, we assume that's the host architecture and simply return it? What if its not the case?